x-pack/platform/plugins/shared/actions/server/config.ts

```diff
@@ -81,6 +81,23 @@ const enabledConnectorTypesSchema = schema.arrayOf(
   }
 );
 
+const rateLimiterSchema = schema.recordOf(
+  schema.string({
+    validate: (value) => {
+      const validConnectorTypeIds = new Set(['email']);
+      if (!validConnectorTypeIds.has(value)) {
+        return `Rate limiter configuration for connector type "${value}" is not supported. Supported types: ${Array.from(
+          validConnectorTypeIds
+        ).join(', ')}`;
+      }
+    },
+  }),
+  schema.object({
+    lookbackWindow: schema.duration({ defaultValue: '15m' }),
```

Comment:
we've been moving away from using `schema.duration` in favor of `schema.string` with duration validation because I believe `schema.duration` accepts the `PT1M` format